packages/suite/src/support/extraDependencies.ts

```diff
@@ -20,13 +25,12 @@ export const extraDependencies: ExtraDependencies = {
         selectAccountTransactions: (state: AppState) => state.wallet.transactions.transactions,
     },
     actions: {
-        addTransaction: transactionActions.add,
-        removeTransaction: transactionActions.remove,
         setAccountLoadedMetadata: metadataActions.setAccountLoaded,
         setAccountAddMetadata: metadataActions.setAccountAdd,
     },
     actionTypes: {
         storageLoad: STORAGE.LOAD,
+        fiatRateUpdate: FIAT_RATES.TX_FIAT_RATE_UPDATE,
```

Comment:
You don't need to this through actionTypes and reducers, please convert fiatRateUpdate action to createAction syntax and then you will have typed payload for that case in common transactionReducer. Exactly same as @bouchja1 did for that two transaction actions before.